packages/aws-cdk/package.json

```diff
@@ -39,6 +39,7 @@
     "@types/request": "^2.47.1",
     "@types/uuid": "^3.4.3",
     "@types/yargs": "^8.0.3",
+    "@types/semver": "^5.5.0",
```

Comment:
You add a dependency on `@types/semver` but not on `semver`... That make sit look fishy.

Reply:
And `semver` is used